client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -305,22 +306,32 @@ export const PatternLibrary = ( {
 									} }
 									value={ patternTypeFilter }
 								>
-									<ToggleGroupControlOption
-										className="pattern-library__toggle-option"
-										label={ translate( 'Patterns', {
-											comment: 'Refers to block patterns',
-											textOnly: true,
-										} ) }
-										value={ PatternTypeFilter.REGULAR }
-									/>
-									<ToggleGroupControlOption
-										className="pattern-library__toggle-option"
-										label={ translate( 'Page Layouts', {
-											comment: 'Refers to block patterns that contain entire page layouts',
-											textOnly: true,
-										} ) }
-										value={ PatternTypeFilter.PAGES }
-									/>
+									<Tooltip
+										text={ translate(
+											'A collection of blocks that make up one section of a page. (For example: an image gallery or contact form)'
```

Comment:
```suggestion
											'A collection of blocks that make up one section of a page.'
```
Seeing this in action, I think it gets problematic with the long text. _Image gallery or contact form_ also sounds quite generic when rendered within the app, since we should technically know which category the user is looking at.

Reply:
Let's also drop the trailing dot to minimize visual noise. It's not clear to me if we should also use `makes` over `make` but since the label is plural I would take advantage of that:
```suggestion
											'Collections of blocks that make up one section of a page'
```

Reply:
> Image gallery or contact form also sounds quite generic when rendered within the app

I agree. I've removed that.

> Let's also drop the trailing dot to minimize visual noise

Dropped